install.sh

```diff
@@ -1001,17 +1001,12 @@ case "${SHELL}" in
     ;;
 esac
 
-# `which` is a shell function defined above.
-# shellcheck disable=SC2230
-if [[ "$(which brew)" != "${HOMEBREW_PREFIX}/bin/brew" ]]
-then
-  cat <<EOS
+cat <<EOS
 - Run these three commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
     echo '# Set PATH, MANPATH, etc., for Homebrew.' >> ${shell_profile}
     echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}
```

Comment:
The existing recommendation still breaks scripts that don't end in a new line (which I've found is typically a consequence of automated editing via some other installer, so the affected user has no idea what the problem is). Maybe we can do something like this:
```suggestion
- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
    printf '\\neval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"\\n' >> ${shell_profile}
```
or, alternatively:
```suggestion
- Run these three commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
    echo ' # Set PATH, MANPATH, etc., for Homebrew.' >> ${shell_profile}
    echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}
```

Reply:
Or how about we always add an extra newline to start with?

Reply:
Yeah, I'll go with @carlocab's first suggestion.